python_http_client/client.py

```diff
@@ -143,18 +143,20 @@ def _build_client(self, name=None):
                       url_path=url_path,
                       append_slash=self.append_slash)
 
-    def _make_request(self, opener, request):
+    def _make_request(self, opener, request, timeout=4):
```

Comment:
This seems incomplete as I don't see a way to get a timeout value down into this call from the public api. It looks to me like the Client.__getattr__ method's nested http_request should take a timeout parameter from the **kwargs and then pass that on to the call to self._make_request. You're hard coding a 4 second timeout here which isn't right - that should be up to the caller. The default timeout shouldn't be set to anything either to stay consistent with how every other api I've used handles timeouts.